### ISSUE TITLE
 Add section or example on mismatching gradle dependencies

### ISSUE THREAD

**Issue by tobrun** [mapbox/android-docs#137]
A user reported the following on the gl-native issue tracker:

```
I think this should deserve a part in the doc. I am not sure how to generalize this, 
but I have struggled for hours on this while updating the cordova mapbox deps to 5.0.1

This snippets worked like a charm:

compile ('com.mapbox.mapboxsdk:mapbox-android-sdk:3.2.0@aar'){
    transitive=true
    exclude group: 'com.android.support', module: 'design'
    exclude group: 'com.android.support', module: 'appcompat-v7'
    exclude group: 'com.android.support', module: 'support-v4'
    exclude group: 'com.android.support', module: 'recyclerview-v7'
}
```

additionally on above we could include some information on `./gradlew :app_module:dependencies` which is a handy tool to troubleshoot these type of configurations.

**Comment by cammace** [comment]
cc: @langsmith

**Comment by langsmith** [comment]
@tobrun / @cammace does this still need to be addressed or is it irrelevant now?

**Comment by langsmith** [comment]
@LukasPaczos @tobrun , do you two think this is still a valid issue? Or can it be closed now that we're on `6.5.0`?

**Comment by tobrun** [comment]
I feel this is still valid, users can still run into this issue
